packages/groff/build.sh

```diff
@@ -0,0 +1,10 @@
+TERMUX_PKG_HOMEPAGE=https://www.gnu.org/software/groff/
+TERMUX_PKG_DESCRIPTION="typesetting system that reads plain text mixed with formatting commands and produces formatted output"
+TERMUX_PKG_LICENSE="GPL-3.0"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION=1.22.4
+TERMUX_PKG_SRCURL=https://ftp.gnu.org/gnu/groff/groff-${TERMUX_PKG_VERSION}.tar.gz
+TERMUX_PKG_SHA256=e78e7b4cb7dec310849004fa88847c44701e8d133b5d4c13057d876c1bad0293
+TERMUX_PKG_DEPENDS="perl"
+TERMUX_PKG_BUILD_DEPENDS="netpbm, psutils, libice, libsm, libxau, libxdmcp, libxcb"
```

Comment:
Remove netpbm, psutils and x11 packages from there. `gxditview` is not a critical functionality also switching to `man-db` from `mandoc` is not planned currently.